LangaraBeetles/beetles-mobile
# feat: Challenge details screen

**Styled Details screen integrated with API**
* ![image](https://github.com/LangaraBeetles/weaup-react-native-app/assets/26020262/a630e4b7-7e03-4b94-8347-65267c3b5349)

**Steps to test:**
1. Make sure to run the API locally. Use the code from [this branch](https://github.com/LangaraBeetles/weaup-api/tree/CU-86b11cydp_Challenge-Detail-and-Share-UI).
2. Run http://localhost:3000/api/v1/auth on postman to get token
3. Copy the token and paste on api.js file 
![image](https://github.com/LangaraBeetles/weaup-react-native-app/assets/26020262/d0a7f59d-54be-4410-b9bd-57efc638aed8)
4. Update your config.ts file. Set the **api_url_local** value to 'http://**[your IP]**:3000/api/v1/' to connect to your local API.


**The following items will be worked on on a separate PR:**
* Screen header --> Create a new component to fix styling and to make the color customizable based on user's selection on create challenge
* Challenge icon --> Create a new component. This should be dynamic based on user's selection on create challenge. Will fix layout
* Member icons --> Create a new component or change to round profile icons with first letter of user's name
